tokens/src/core/global/typography.json

```diff
@@ -11,7 +11,16 @@
         "sans": {
           "serif": {
             "source": "$font-family-sans-serif",
-            "$value": "-apple-system, BlinkMacSystemFont, 'Segoe UI',  Roboto, 'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji',  'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji'",
+            "$value": {
+              "primary": "-apple-system",
+              "secondary": "BlinkMacSystemFont",
+              "tertiary": "'Segoe UI'",
+              "alternate-1": "Roboto",
+              "alternate-2": "'Helvetica Neue'",
+              "alternate-3": "Arial",
+              "alternate-4": "'Noto Sans'",
+              "fallback": "sans-serif"
+            },
```

Comment:
It seems this no longer has the following
```
'Apple Color Emoji',  'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji'
```

Should we add all of those as additional alternates? Is there a better way to handle this?

Reply:
After reviewing the Token Studio library documentation, I noticed that for properties like box shadow, there is an interface that allows users to adjust each value of the composite shadow token (e.g., x-axis, y-axis, etc.). In the context of the shadows provided by Paragon, this makes perfect sense, and consumers will be able to fully utilize this functionality.

<img width="1920" alt="tokens-boxShadow-form-empty-2-01" src="https://github.com/user-attachments/assets/35385795-ff8c-4218-8f0c-8c70064c3a32">

Similarly, on [the Typography compositional tokens](https://docs.tokens.studio/token-types/token-type-typography-composite) description page, we can see that fonts are also composed of several elements, and an interface is available for editing them.

<img width="1920" alt="tokens-typography-form-annotated-2-01" src="https://github.com/user-attachments/assets/6affa4e5-f574-41dd-ac8b-038c60da3be2">

However, I observed that for the fontFamily property, there is only a single selection field without a clear distinction between alternative fonts and fallbacks. Should we consider splitting the fontFamily property into separate tokens for each font (e.g., primary, alternative, fallback), or does that approach not add value? I think we should keep the fontFamily values ​​as a single (non-divisible) value. What are your thoughts?

Reply:
I'd be fine with keeping it as one long string with not splitting it out and keeping what we had before, so just setting the `fontFamily` value to
```
"-apple-system, BlinkMacSystemFont, 'Segoe UI',  Roboto, 'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji',  'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji'"
```

I'd like to hear @adamstankiewicz's thoughts on this though